balancer-js/src/modules/pools/pool-types/concerns/weighted/join.concern.integration.spec.ts

```diff
@@ -1,22 +1,16 @@
 /* eslint-disable no-unexpected-multiline */
```

Comment:
@johngrantuk comment:
Can you add a test for joining with ETH please.
Also there are Weighted V3 pools on mainnet now. Can we add tests for those too please. I think its best to create tests for different versions in different files similar to the ComposableStable PR additions I added. I think its a bit easier to follow then.

Reply:
test joining with ETH: Done;
tests with Weighted V3: Couldn't get expected values in balanceDeltas, I think it's related with protocol fees 😕